src/ccl/charts/chart.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, ViewEncapsulation, Inject } from '@angular/core';
+import { ChartService } from '../services/chart.service';
+import { LineGraph } from './line-graph.component';
+
+/*
+ * Chart component
+ * Management for the sections that appear in the chart container
+ */
+
+@Component({
+  selector: 'charts',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
I'm not sure using this encapsulation method is in general a good idea. It feels awkward and breaks the idea that a component folder has all of the stuff necessary to use it. Does our webpack config need additional work to support the default 'emulated' mode?